core/src/test/scala/unit/kafka/server/LogOffsetTest.scala

```diff
@@ -112,7 +112,7 @@ class LogOffsetTest extends ZooKeeperTestHarness {
   @Test
   def testEmptyLogsGetOffsets() {
     val topicPartition = "kafka-" + random.nextInt(10)
-    val topicPartitionPath = getLogDir.getAbsolutePath + "/" + topicPartition
+    val topicPartitionPath = ZkUtils.getZkPath(getLogDir.getAbsolutePath, topicPartition)
```

Comment:
This looks weird. `topicPartitionPath` is probably log path on disk. We should not be using `ZkUtils.getZkPath` here, right?